### DISCUSSION TITLE
How to create a PDF/A for long term archiving?

### DISCUSSION THREAD

**Discussion by marbetschar** [LibrePDF/OpenPDF#762] [Q&A]
How do I create a document in PDF/A format using OpenPDF? The information I'm able to find in the web is based upon iText and some classes which do not seem to be available in OpenPDF.

Any help would be highly appreciated.

**Comment by andreasrosdalw** [comment]
Please see #86. I don't think PDF/A is supported yet. OpenPDF is an open source, volunteer-based project, so feel free to help drive this feature forward. Pull-requests welcome!

**Comment by marbetschar** [comment]
@andreasrosdal thanks for your feedback. I really love OpenPDF and all the work you putting in, to drive it forward as the open source successor of iText!

I'm aware that some features might not be implemented yet - but I was curious if I missed something in regards of PDF/A. PDF/A is not a requirement here as of now. So, unfortunately, I don't have the time to dig deeper into this. However it would be a nice addition to create future proof PDF files.

**Comment by sabraMa** [comment]
Hello
Do you have an idea if this functionality is implemented int the newest version?

**Comment by SHRIVP** [comment]
Hi,

Is this feature to convert to pdf/a available now?

Thanks,
Pushkar

**Comment by asturio** [comment]
I don't think so. But Pull Requests are welcome :-).

**Comment by andreasrosdalw** [comment]
Please see #980
